libpsautohint/autohintexe.c

```diff
@@ -436,6 +436,7 @@ main(int argc, char* argv[])
             }
 
             ACBufferFree(output);
+            output=NULL;
```

Comment:
It might be a good idea to make a, say, `AC_BUFFER_FREE` macro that calls `ACBufferFree()` and then do the `NULL` assignment, and use it everywhere `ACBufferFree()` is currently used.

Reply:
I see can the advantages of that, but given the debugging pain I went through to get to this point, I'd really rather be able to search on pointers and see where they are set without having to remember what a macro does.

Reply:
Also, just to be clear on what this point is: I think I've fixed the crashes, and now we just need to sort out the output differences. I've asked @readroberts to take a look at the output.